apps/builder/app/canvas/shared/use-track-selected-element.ts

```diff
@@ -38,6 +40,12 @@ export const useTrackSelectedElement = () => {
 
   useEffect(() => {
     const handleClick = (event: MouseEvent) => {
+      // When user is selecting text inside content editable and mouse goes up
+      // on a different instance - we don't want to select a different instance
+      // because that would cancel the text selection.
+      if (isHighlighting()) {
```

Comment:
Why click on other instance after selecting text in editor?

Reply:
see video: https://github.com/webstudio-is/webstudio-builder/issues/1193

you start selecting in our instance, then mouse-up happens over another instance